specs/rollup-node-p2p.md

```diff
@@ -276,7 +276,7 @@ An [extended-validator] checks the incoming messages as follows, in order of ope
 
 - `[REJECT]` if the compression is not valid
 - `[REJECT]` if the block encoding is not valid
-- `[REJECT]` if the `payload.timestamp` is older than 20 seconds in the past
+- `[REJECT]` if the `payload.timestamp` is older than 60 seconds in the past
```

Comment:
did the value change in the code in this PR?

Reply:
If L1 block time is 15 seconds, and there's some response from the sequencer (e.g. a reorg or some other stalling), it's likely to produce L2 blocks with a delay. So I bumped it up to be more flexible.